docs/decisions/adr.md.template

```diff
@@ -0,0 +1,18 @@
+# {YYYY-MM-DD} {short title of solved problem and solution}
```

Comment:
Thanks for getting this set up -- super excited to start seeing these in our codebase!! 🥳